include/tester/vtcp/tcp/TCPSession.hpp

```diff
@@ -18,7 +20,11 @@ class TCPSession
 public:
     TCPSession(tcp::socket socket, const std::size_t id,
     std::function<void(const std::size_t, const Message&)> onMessageReceived,
-    std::function<void(const std::size_t)> onDisconnect) : 
+    std::function<void(const std::size_t)> onDisconnect,
+    std::function<bsoncxx::document::value(int)> startAutoTest,
```

Comment:
This class doesn't need access to any of the mongodb library classes. its a generic TCPSession class for a sole purpose - to provide tcp support.

Reply:
So I will save those functions in VTCPSession?

Reply:
done